source/fundamentals/enterprise-authentication.txt

```diff
@@ -2,4 +2,264 @@
+**authentication mechanisms** available only in the {+mongo-enterprise+}.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version of
+{+mongo-enterprise+}.
+
+- :ref:`csharp-kerberos`
+- :ref:`csharp-LDAP`
+
+To authenticate using another mechanism, see the
+:ref:`<csharp-authentication-mechanisms>` fundamentals page. For
+more information on establishing a connection to your MongoDB cluster, see the
+:ref:`csharp-connect-to-mongodb`.
+
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using either of the following methods:
+
+- A **connection string**, also known as a **connection URI**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- A factory method for the supported authentication mechanism, contained in the 
+  ``MongoCredential`` class.
+
+.. _csharp-kerberos:
+
+Authenticate with GSSAPI/Kerberos
+---------------------------------
+
+The Generic Security Services API (GSSAPI) authentication mechanism allows the 
+user to authenticate to a Kerberos service using the user's principal name.
+
+The following examples specify the authentication mechanism using the following
+placeholders:
+
+- ``<username>``: Your :wikipedia:`URL-encoded <Percent-encoding>` principal name; for example "username%40REALM.ME"
+- ``<password>``: Your Kerberos user's password
+- ``<hostname>``: The network address of your MongoDB server, accessible by your client
+
+Select the :guilabel:`Connection String` or :guilabel:`MongoCredential` tab to
+see the corresponding syntax for specifying the GSSAPI/Kerberos authentication
+mechanism:
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connection-string
+
+      .. code-block:: csharp
+
+         var mongoClient = new MongoClient("mongodb://<username>:<password>@<hostname>/?authMechanism=GSSAPI");
+
+   .. tab:: MongoCredential
+      :tabid: mongo-credential
+
+      .. code-block:: csharp
+         :emphasize-lines: 1
+
+         var credential = MongoCredential.CreateGssapiCredential("<username>", "<password>");
+         var settings = MongoClientSettings.FromConnectionString("<connection string>");
+         settings.Credential = credential;
+         var mongoClient = new MongoClient(settings);
+
+.. tip:: Omitting the Password
+
+   You can omit the password if one of the following are true:
+
+   - On Windows, the process owner running the application is the same as the
+     user needing authentication.
+
+   - ON Linux, the user has initialized their keytab via ``kinit username@REALM.COM``.
+  
+Additional Properties
+~~~~~~~~~~~~~~~~~~~~~
+
+You can specify additional properties when specifying your authentication
+mechanism, using the connection string or a factory method in the ``MongoCredential``
+class.
```

Comment:
Is there a way to avoid using "specify/specifying" twice in one sentence? (I think we can lose the comma before "using" too)